yscope-compose/README.md

```
@@ -0,0 +1,76 @@
+# You probably want to run a python 3.9 or newer virtual environment
+sbin/start-clp.sh
+```
+
+5. Compress:
+
+```bash
+# You can also use your own dataset
+sbin/compress.sh --timestamp-key 'timestamp' /path/to/postgresql.log
+```
+
+6. Use the following command to update `.env`:
```

markdownlint-cli2 (0.17.2)

Ordered list item prefix
Expected: 1; Actual: 6; Style: 1/1/1

(MD029, ol-prefix)